src/libraries/controllers/ApiUserController.php

```diff
@@ -55,7 +55,7 @@ public function passwordRequest()
     $email = $_POST['email'];
     if($email == $this->config->user->email)
     {
-      $token = md5(rand(10000,100000));
+      $token = md5(rand(10000,10000000));
```

Comment:
lgtm

Reply:
I recommend using http://php.net/manual/de/function.openssl-random-pseudo-bytes.php for this.